### DISCUSSION TITLE
clickhouse数据库乱码

### DISCUSSION THREAD

**Discussion by goobycle** [deepflowio/deepflow#5935] [Q&A]
in_process表profile_location_str字段乱码，如(�/�1[p] deepflow-agent;[t] cgroups-control�>k，如何调整，烦请支持。

**Comment by goobycle** [comment]
求回复。

**Comment by tjlqq** [comment]
SELECT trace_id  from flow_log.l7_flow_log where  l7_protocol_str =  'MySQL' and request_type = 'COM_QUERY' and toUInt32(time) = 1720594701   这个sql查出来也是乱码；
![Uploading 企业微信截图_17206013422612.png…]()
